[2024-05-02T08:23:31.132+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-02T08:23:31.154+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: s3-data-transfer-dag.transfer-customer manual__2024-05-02T08:23:29.635875+00:00 [queued]>
[2024-05-02T08:23:31.162+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: s3-data-transfer-dag.transfer-customer manual__2024-05-02T08:23:29.635875+00:00 [queued]>
[2024-05-02T08:23:31.162+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-05-02T08:23:31.173+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): transfer-customer> on 2024-05-02 08:23:29.635875+00:00
[2024-05-02T08:23:31.181+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=684) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-02T08:23:31.182+0000] {standard_task_runner.py:63} INFO - Started process 689 to run task
[2024-05-02T08:23:31.183+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 's3-data-transfer-dag', 'transfer-customer', 'manual__2024-05-02T08:23:29.635875+00:00', '--job-id', '73', '--raw', '--subdir', 'DAGS_FOLDER/new-dag.py', '--cfg-path', '/tmp/tmpqhb0b833']
[2024-05-02T08:23:31.184+0000] {standard_task_runner.py:91} INFO - Job 73: Subtask transfer-customer
[2024-05-02T08:23:31.222+0000] {task_command.py:426} INFO - Running <TaskInstance: s3-data-transfer-dag.transfer-customer manual__2024-05-02T08:23:29.635875+00:00 [running]> on host 0a71bd2d4f31
[2024-05-02T08:23:31.288+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='s3-data-transfer-dag' AIRFLOW_CTX_TASK_ID='transfer-customer' AIRFLOW_CTX_EXECUTION_DATE='2024-05-02T08:23:29.635875+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-02T08:23:29.635875+00:00'
[2024-05-02T08:23:31.289+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-02T08:23:32.021+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-02T08:23:32.022+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-02T08:23:32.029+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=s3-data-transfer-dag, task_id=transfer-customer, execution_date=20240502T082329, start_date=20240502T082331, end_date=20240502T082332
[2024-05-02T08:23:32.079+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-02T08:23:32.093+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-02T08:23:32.094+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
